internal/characters/amber/bunny.go

```diff
@@ -108,12 +236,11 @@ func (c *char) overloadExplode() {
 		if atk.Info.AttackTag != attacks.AttackTagOverloadDamage {
 			return false
 		}
+		c.bunnies[0].ae.Info.Abil = manualExplosionAbil
```

Comment:
Shouldn't this be set for all the bunnies exploded here? So do like `v.ae.Info.Abil = manualExplosionAbil` as part of the loop below.

Reply:
I'm not sure. The original code did this and there is no evidence to the contrary so I didn't change it.

Reply:
I think it should be set for all bunnies.